ext/opcache/zend_accelerator_module.c

```diff
@@ -370,12 +371,48 @@ static ZEND_NAMED_FUNCTION(accel_is_readable)
 	}
 }
 
+void zend_jit_validate_opcache_attribute(zend_attribute *jit, uint32_t target, zend_class_entry *scope)
+{
+	if (jit->argc > 1) {
+		zend_error(E_COMPILE_ERROR, "@@Jit requires zero or one argument, %d arguments given", jit->argc);
```

Comment:
It seems inconsistent to emit E_COMPILE_ERROR with opcache but not without opcache. I'd personally prefer E_WARNING like some of the other warnings about failing to optimize.

- And to return early but proceed with optimizing the file anywhere where the attribute is seen with invalid count or argument types 

```
ext/opcache/zend_accelerator_module.c
893:                    zend_error(E_WARNING, ACCELERATOR_PRODUCT_NAME " could not compile file %s", handle.filename);
```

Also, `zend_error` proceeds with execution after being called. You may want `zend_error_noreturn(E_COMPILE_ERROR, ` if the intent is actually to make php stop executing - I don't know if that actually works in the opcache step, though